core/object.cpp

```diff
@@ -1723,7 +1723,7 @@ void Object::_bind_methods() {
 	BIND_VMETHOD(MethodInfo("_set", PropertyInfo(Variant::STRING, "property"), PropertyInfo(Variant::NIL, "value")));
 #ifdef TOOLS_ENABLED
 	MethodInfo miget("_get", PropertyInfo(Variant::STRING, "property"));
-	miget.return_val.name = "var";
+	miget.return_val.name = "Variant";
```

Comment:
I don't know about that but `var` or `Variant` , the documentation shows `void` as the return type and `return_val.name = "Variant"` is already in use in more places.

![screenshot](https://user-images.githubusercontent.com/10124094/28363926-eb83b410-6c71-11e7-84da-323203c443a0.png)